### PR TITLE
Update versions - 2019-05-08

### PR DESCRIPTION
**Pull request by ovk6**
Separated default configuration for 6 and 7 because configuration setting `elasticsearch.url` is deprecated and is replaced by `elasticsearch.hosts`
Source: https://www.elastic.co/guide/en/kibana/current/settings.html

### CODE REVIEW

File: versions.yaml [hunk 1]
```diff
@@ -16,16 +16,79 @@ _variables:
...
+      - '6.7'
+    from: *from
+    packages:
+      kibana:
+        version: '6.7.2'
+        sha512: '71b89f326374319d526e7fa6af916463333990c3723150f5cc55450a239adc6f4202e9fb9924e09accd7e956721c2795c1652713cd784da3f78bf26ac9dc47bb'
+  - dir: '6/debian9/6.6'
+    repo: kibana6
+    tags:
+      - '6.6.2-debian9'
```
Comment (wgrzelak) [inline versions.yaml]: Do we really need to build these all versions?

Comment (ovk6) [reply versions.yaml]: Our application uses 6.3 and only this version is tracked on Marketplace. But for some reason rotation script had also hardcoded 6.4 and 6.5 versions. So I added all more recent versions after 6.3.

If it is more appropriate we can stick to 6.3 + latest (which is 6.8 for Kibana 6).

Also should we add Kibana 7?

Comment (ovk6) [reply versions.yaml]: As discussed offline, leaving only 6.3, 6.8, 7.1

File: versions.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Google LLC
+# Copyright 2019 Google Inc.
```
Comment (wgrzelak) [inline versions.yaml]: Please revert to LLC.

```suggestion
# Copyright 2019 Google LLC
```

**Comment by wgrzelak** [comment]
The kibana doesn't have functional_tests. Have you make sure that new versions, especially 7.1 works?

File: versions.yaml (hunk 2 above)
Comment (ovk6) [reply versions.yaml]: Done

**Comment by ovk6** [comment]
Yes, I made sure it works.
The configuration was separated as I said in PR description.

I believe the tests are not implemented because it is not a standalone app, and requires elasticsearch server to verify it is working.

**Pull request merged**